sdc/tests/test_dataframe.py

```diff
@@ -1143,15 +1143,46 @@ def test_impl(A):
         hpat_func = self.jit(test_impl)
         pd.testing.assert_frame_equal(hpat_func(df), test_impl(df2))
 
-    @skip_numba_jit
-    def test_df_reset_index1(self):
+    def test_df_reset_index_drop_True(self):
```

Comment:
Can we extend this test with different indexes (each tested in separate subTest) in the DF?
The indexes can be:
indexes_to_test = {
     'default': None,
     'int': np.arange(n),
     'float': np.arange(n, dtype='float'),
     'str': gen_strlist(n, 2, 'abcd123 ')
}